### PR TITLE
[TS] LPP-46038 > LPS-1587895 Trimming the url before testing

### PR DESCRIPTION
**Pull request by ces-totrinh**
Hi @tinatian,
A unit test has been added based on your comment on the previous PR(https://github.com/liferay-core-infra/liferay-portal/pull/1152). Please help review these changes.
Tickets:
https://issues.liferay.com/browse/LPP-46038
https://issues.liferay.com/browse/LPS-158795
Regards,
cc: @hongvo2308

### CODE REVIEW

**Review by dantewang — CHANGES_REQUESTED** [review]
@tototrinh Please update the PR according to my comments; also, please consider your test case, because at the first glance I don't understand why this should be converted to `null`. Is it possible to have a better pair of input/expected?

BTW, there's a typo in your first commit (ticket number).

File: portal-impl/test/unit/com/liferay/portal/util/PortalImplUnitTest.java [hunk 1]
```diff
@@ -52,6 +52,13 @@ public class PortalImplUnitTest {
 	public static LiferayUnitTestRule liferayUnitTestRule =
 		LiferayUnitTestRule.INSTANCE;
 
+	@Test
+	public void testEscapeRedirectWithEscapingSequenceCharacter() {
+		String url = "\t//example.com";
```
[the remark this answers is not in the document]
Comment (tinatian) [reply portal-impl/test/unit/com/liferay/portal/util/PortalImplUnitTest.java:57]: @tototrinh Please add the test to PortalImplEscapeRedirectTest.

**Comment by tinatian** [comment]
@tototrinh Please see comments and resend.

Thanks.
Tina.

**Comment by hongvo2308** [comment]
Hi @dantewang 
About your concerns when it return null. I think it's from the original implementation at https://github.com/tototrinh/liferay-portal/blob/4bfc5b7987ddeba9c921d5d58b1fa2ae416b4272/portal-impl/src/com/liferay/portal/util/PortalImpl.java#L932-L946. So this implementation is mainly about trimming the escaping sequence character before the check.
Thank you.
